src/components/TodosList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Todo } from '../types/todo';
+import { TodoListItem } from './TodoListItem';
+
+type Props = {
+  todos: Todo[]
+};
+type State = {};
+
+export class TodosList extends React.Component<Props, State> {
+  state = {};
```

Comment:
```suggestion
export class TodosList extends React.Component<Props,{}> {
```
Можна і не вказувати стейт, якщо не потрібно